README.md

```diff
@@ -1,2 +1,10 @@
 # baritonesWebsite
-Website for the Bearded Baritones.  This website is live at http://beardedbaritones.com.  
+Website for the Bearded Baritones.  This website is live at http://beardedbaritones.com.
+
+To run the server:
+1) Install node.js from nodejs.org
+2) Get the secret config file, and place it in config/index.js
+3) Install the server:
```

Comment:
This command doesn't really install the server.  It installs the dependencies for the project.